wai-aria/scripts/aria-utils.js

```diff
@@ -136,10 +136,22 @@ const AriaUtils = {
       let label = el.getAttribute("data-expectedlabel");
       let testName = el.getAttribute("data-testname") || label; // data-testname optional if label is unique per test file
       promise_test(async t => {
-        const expectedLabel = el.getAttribute("data-expectedlabel");
+        let expectedLabel = el.getAttribute("data-expectedlabel");
         let computedLabel = await test_driver.get_computed_label(el);
-        // Todo: Remove whitespace normalization after https://github.com/w3c/accname/issues/192 is addressed. Change prior line back to `const`, too.
-        computedLabel = computedLabel.trim()
+
+        // See:
+        // - https://github.com/w3c/accname/pull/165
+        // - https://github.com/w3c/accname/issues/192
+        // - https://github.com/w3c/accname/issues/208
+        //
+        // AccName reference's HTML's definition of ASCII Whitespace
+        // https://infra.spec.whatwg.org/#ascii-whitespace
+        // which matches tab (\t), newline (\n), formfeed (\f), return (\r), and regular space (\u0020).
+        // but it does NOT match non-breaking space (\xA0,\u00A0) and others matched by \s
+        const asciiWhitespace = /\t\n\f\r\u0020+/g;
+        expectedLabel = expectedLabel.replace(asciiWhitespace, '\u0020').trim();
```

Comment:
```suggestion
```

Reply:
In hindsight, it seems obvious that I should not be normalizing trimming the expectation, too! 🤣